### PR TITLE
Add translation for ReplicationController in ID localization

### PR DESCRIPTION
**Pull request by joshuabezaleel**
This address #15341 for ReplicationController translation to Bahasa Indonesia (id) localization.

### CODE REVIEW

File: content/id/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+Sebuah ReplicationController juga membutuhkan [bagian `.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+### Templat Pod
+
+`.spec.template` adalah satu-satunya _field_ yang diwajibkan pada `.spec`.
+
+`.spec.template` adalah sebuah [templat pod](/docs/concepts/workloads/pods/pod-overview/#pod-templates). Ia memiliki skema yang sama persis dengan sebuah [pod](/docs/concepts/workloads/pods/pod/), namun dapat berbentuk _nested_ dan tidak memiliki _field_ `apiVersion` ataupun `kind`.
+
+Selain _field-field_ yang diwajibkan untuk sebuah Pod, template pod pada ReplicationControll harus menspesifikasi label dan kebijakan pengulangan kembali yang tepat. Untuk label, pastikan untuk tidak tumpang tindih dengan kontroler lain. Lihat [selektor pod](#selektor-pod).
```
[the remark this answers is not in the document]
Comment (phanama) [reply content/id/docs/concepts/workloads/controllers/replicationcontroller.md]: Let's use Selektor if it's comprehensible enough for the readers.
Reference:
https://github.com/kubernetes/website/pull/19072#discussion_r395456792

**Comment by phanama** [comment]
Hello @joshuabezaleel 👋 
Any updates?

**Comment by joshuabezaleel** [comment]
Hi @phanama @girikuncoro , truly sorry that I missed this review. Thank you very much @phanama for the really thorough review, truly appreciate it! :slightly_smiling_face: I've pushed the changes.

File: content/id/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) adalah objek API tingkat tinggi yang memperbarui ReplicaSet dan Pod-Pod di bawahnya yang mirip dengan cara kerja `kubectl rolling-update`. Deployment direkomendasikan jika kamu menginginkan fungsionalitas dari pembaruan bergulir ini, karena tidak seperti `kubectl rolling-update`, Deployment memiliki sifat deklaratif, _server-side_, dan memiliki beberapa fitur tambahan lainnya.
+
+### Pod sederhana
+
+Tidak seperti pada kasus ketika pengguna secara langsung membuat Pod, ReplicationController menggantikan Pod-Pod yang dihapus atau dimatikan untuk alasan apapun, seperti pada kasus kegagalan Node atau pemeliharaan Node yang disruptif, seperti pembaruan kernel. Untuk alasan ini, kami merekomendasikan kamu untuk menggunakan ReplicationController bahkan ketika aplikasimu hanya membutuhkan satu Pod saja. Anggap hal ini mirip dengan pengawas proses, hanya pada kasus ini mengawasi banyka Pod yang terdapat pada berbagai Node dan bukan proses-proses tunggal pada satu Node. ReplicationController mendelegasikan pengulangan kontainer lokal ke agen yang terdapat dalam Node (contohnya Kubelet atau Docker).
```
Comment (girikuncoro) [inline content/id/docs/concepts/workloads/controllers/replicationcontroller.md]: typo
```suggestion
Tidak seperti pada kasus ketika pengguna secara langsung membuat Pod, ReplicationController menggantikan Pod-Pod yang dihapus atau dimatikan untuk alasan apapun, seperti pada kasus kegagalan Node atau pemeliharaan Node yang disruptif, seperti pembaruan kernel. Untuk alasan ini, kami merekomendasikan kamu untuk menggunakan ReplicationController bahkan ketika aplikasimu hanya membutuhkan satu Pod saja. Anggap hal ini mirip dengan pengawas proses, hanya pada kasus ini mengawasi banyak Pod yang terdapat pada berbagai Node dan bukan proses-proses tunggal pada satu Node. ReplicationController mendelegasikan pengulangan kontainer lokal ke agen yang terdapat dalam Node (contohnya Kubelet atau Docker).
```

Comment (girikuncoro) [reply content/id/docs/concepts/workloads/controllers/replicationcontroller.md]: banyka -> banyak

File: content/id/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+ReplicationController akan selalu dibatasi terhadap tanggung jawab spesifik ini. Ia tidak akan melakukan _probe_ kesiapan atau keaktifan. Daripada melakukan _auto-scaling_, ia ditujukan untuk dikontrol oleh _auto-scaler_ eksternal (seperti yang didiskusikan pada [#492](http://issue.k8s.io/492)), yang akan mengganti _field_ `replicas`. Kami tidak akan menambahkan kebijakan penjadwalan (contohnya [_spreading_](http://issue.k8s.io/367#issuecomment-48428019)) untuk ReplicationController. Ia juga tidak seharusnya melakukan verifikasi terhadap Pod-POd yang sedang dikontrol yang cocok dengan spesifikasi templat saat ini, karena hal itu dapat menghambat _auto-sizing_ dan proses otomatis lainnya. Demikian pula batas waktu penyelesaian, pengurutan _dependencies_, ekspansi konfigurasi, dan fitur-fitur lain yang seharusnya berada di komponen lain. Kami juga bahkan berencana untuk mengeluarkan mekanisme pembuatan Pod secara serentak ([#170](http://issue.k8s.io/170)).
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/concepts/workloads/controllers/replicationcontroller.md]: Pod-POd -> Pod-Pod

File: content/id/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+
+### Job
+
+Gunakan [`Job`](/docs/concepts/jobs/run-to-completion-finite-workloads/) sebagai ganti ReplicationController untuk Pod-Pod yang diharapkan diterminasi dengan sendirinya (seperti _batch jobs_).
+
+### DaemonSet
+
+Gunakan [`DaemonSet`](/docs/concepts/workloads/controllers/daemonset/) sebagai ganti ReplicationController untuk Pod-POd yang menyediakan fungsi pada level mesin, seperti pengamatan mesin atau pencatatan mesin. Pod-Pod ini memiliki waktu hidup yang bergantung dengan waktu hidup mesin: Pod butuh untuk dijalankan di mesin sebelum Pod-Pod lainnya dimulai, dan aman untuk diterminasi ketika mesin sudah siap untuk dinyalakan ulang atau dimatikan.
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/concepts/workloads/controllers/replicationcontroller.md]: Pod-POd -> Pod-Pod

File: content/id/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+
+Ketika menggunakan REST API atau _library_ klien go, cukup hapus objek ReplicationController.
+
+Ketika ReplicationController yang asli telah dihapus,kamu dapat membuat ReplicationController yang baru sebagai penggantinya. Selama `.spec.selector` yang lama dan baru memiliki nilai yang sama, maka ReplicationController baru akan mengadopsi Pod-Pod yang lama.
+Walaupun begitu, ia tidak akan melakukan usaha apapun untuk membuat Pod-Pod yang telah ada sebelumnya untuk sesuai dengan templat Pod yang baru dan berbeda.
+Untuk memerbarui Pod-Pod ke spesifikasi yang baru dengan cara yang terkontrol, gunakan [pembaruan bergulir](#pembaruan-bergulir).
+
+### Mengisolasi Pod dari ReplicationController
+
+Pod-Pod dapat dihapus dari kumpulan target sebuah ReplicationController dengan menganti nilai dari labelnya. Teknik ini dapat digunakan untuk mencopot Pod-POd dari servis untuk keperluan pengawakutuan (_debugging_), pemulihan data, dan lainnya. Pod-Pod yang dicopot dengan cara ini dapat digantikan secara otomatis (dengan asumsi bahwa jumlah replika juga tidak berubah).
```
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/concepts/workloads/controllers/replicationcontroller.md]: menganti -> mengganti

**Comment by girikuncoro** [comment]
awesome, thanks @joshuabezaleel! 🎉 

/lgtm
/approve

**Pull request merged**